opm/simulators/wells/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -374,6 +374,18 @@ namespace Opm
             return it->second;
         }
 
+        void setCurrentInjectionGroupReservoirRates(const std::string& groupName, const std::vector<double>& target ) {
+            injection_group_reservoir_rates[groupName] = target;
+        }
+        const std::vector<double>& currentInjectionGroupReservoirRates(const std::string& groupName) const {
```

Comment:
Please add blanks between the functions, especially when there are blanks inside the functions.

Reply:
done